src/tilesource.js

```diff
@@ -538,9 +538,11 @@ $.TileSource.prototype = {
                             formattedExc = exc.toString();
                         }
 
-                        msg = formattedExc + " attempting to load TileSource";
+                        msg = formattedExc + " attempting to load TileSource: " + url;
                     }
 
+                    $.console.error(msg);
```

Comment:
Should we not write to the console here if somebody has subscribed to the `open-failed` event? We may need to add something like `hasHandler` to `EventSource` to make that possible, but it shouldn't be hard.

Reply:
Good idea!

Reply:
Actually, maybe not... The problem is `open-failed` event is registered inside OpenSeadragon to print the message into the error html container, so in 99,9999% scenarios this equals to not logging the message at all...